css/_common.scss

```diff
@@ -1,88 +1,71 @@
 html {
-	height: 100%;
+  height: 100%;
 }
 
 body {
-	min-height: 100%;
-	padding: 1rem;
-	background-color: $color-light;
-	font-family: $font-main;
+  background-color: $color-white;
+  font-family: $font-main;
+  min-height: 100%;
+  padding: 1rem;
 
-	&.maxWidth {
-		height: 100%;
-		margin: 0 auto;
-		max-width: 1200px;
-	}
-}
-
-header {
-	form {
-		background-color: #fff;
-		border-radius: .7rem;
-		@include flex(space-between, center);
+  &.max-width {
+    height: 100%;
+    margin: 0 auto;
 
-		* {
-			height: 3rem;
-			padding: 0;
-			border: 0;
-			margin: .7rem;
-		}
-	}
+    @include breakpoint(md) {
+      max-width: 1200px;
+    }
+  }
 }
 
 .row {
-	@include flex(space-between, center);
-}
+  display: block;
 
-.lists {
-	display: grid;
-	gap: 3%;
-	grid-template-columns: repeat(3, 31%);
+  @include breakpoint(md) {
+    @include flex(space-between, center);
+  }
 
-	.photo {
-		overflow: hidden;
-		border-radius: 1rem;
-		max-height: 250px;
-		aspect-ratio: 16 / 10;
-		position: relative;
-		margin-bottom: 1rem;
+  .col {
+    display: block;
+  }
 
-		img {
-			display: block;
-			width: 100%;
-			height: auto;
-			position: absolute;
-			bottom: 0;
-		}
-	}
+}
+
+header {
+  form {
+    @include flex(space-evenly, center);
+    width: 100%;
+    border-radius: .7rem;
+    background-color: $color-white;
+    padding-block: 1rem;
+    margin-block: 1rem;
+    box-shadow: 0 0.1rem .5rem $color-lighter;
 
-	.details {
-		.unit {
-			padding: .3rem 0;
-			margin-bottom: .5rem;
-			@include flex(space-between, center);
+    @include breakpoint(md) {
+      padding-block: 0;
+      margin-block: 1rem;
+    }
 
-			.host {
-				padding: .5rem;
-				text-transform: uppercase;
-				font-size: 80%;
-				border-radius: .8em;
-				border: 1px solid $color-dark;
-			}
+    select,
+    input {
+      border: 0;
+      padding: 0;
 
-			.type, .bed {
-				color: dimgrey;
-			}
-		}
+      @include breakpoint(md) {
+        height: 3rem;
+        margin: .7rem;
```

Comment:
Following this CSS guideline, it's better to include zero when a decimal mark is needed.

Ref: https://github.com/xvpn/frontier/pull/2542

Reply:
Got it!